src/components/SignIn.js

```diff
@@ -1,49 +1,101 @@
-                <Form>
-                    <h1 className="header-css">Login to your account</h1>
+            <div className = "split left">
+                <div className="centered">
+                    <img className="info-Banner" src={picLogScreen} alt="picLogScreen"/>
+                </div>
+            </div>
+        
+            <div className = "split right">
+                <div className="centered">
+                    <Form>
+                        <h1 className="header-css">Login to Your Account</h1>
           
+                        <Form.Group className="mb-3" controlId="form-Basic-Username">
+                            <Form.Label>Username</Form.Label>
+                            <Form.Control type="username" placeholder="Enter username" onChange={(e) => {setUsername(e.target.value)}}/>
+                        </Form.Group>
   
-                <Form.Group className="mb-3" controlId="form-Basic-Email">
-                    <Form.Label>Email</Form.Label>
-                    <Form.Control type="email" placeholder="Enter email" />
-                </Form.Group>
-  
-                <Form.Group className="mb-3" controlId="form-Basic-Password">
-                    <Form.Label>Password</Form.Label>
-                    <Form.Control
-                        type="password"
-                        placeholder="Password"/>
-                </Form.Group>
+                        <Form.Group className="mb-3" controlId="form-Basic-Password">
+                            <Form.Label>Password</Form.Label>
+                            <Form.Control
+                                type="password"
+                                placeholder="Password" onChange={(e) => {setPassword(e.target.value)}}/>
+                        </Form.Group>
   
-                <Form.Group className="mb-3" controlId="submit-form">
-                    <Button variant="outline-info" type="submit">
-                        Login
-                    </Button> 
-                </Form.Group>
+                        <Form.Group className="mb-3" controlId="submit-form">
+                            <Button variant="outline-info" type="submit" to='/users/:id' onClick={handleClick}>
+                                Login
+                            </Button> 
+                        </Form.Group>
   
-                <Form.Group className="mb-3" controlId="link-to-SignUp">
-                    <Button variant="secondary" type="submit">
-                        Dont have an account?
-                    </Button>
-                </Form.Group>
+                        <Form.Group className="mb-3" controlId="link-to-SignUp">
+                            <Link variant="secondary" type="submit" to='/auth/signup'>
+                                Dont have an account?
+                            </Link>
+                        </Form.Group>
  
-                </Form>
-                </Col>
-            </Row>
+                        {error ? (
```

Comment:
Instead of `?`, this could have been replaced with `&&`, which would remove the need for `null`.